tfhe/src/shortint/server_key/mod.rs

```diff
@@ -276,6 +277,7 @@ impl ServerKey {
             carry_modulus: self.carry_modulus,
             degree: Degree(self.message_modulus.0 - 1),
             pbs_order: self.pbs_order,
+            noise_level: NoiseLevel::NOMINAL,
```

Comment:
Let's open an issue for the conformance to manage NoiseLevel Ranges as valid ciphertexts may have noise levels which are not nominal (like being loaded from disk after some unchecked operations or equivalent), those ciphertexts can very well be valid still

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/300